packages/cx-layout/src/components/Layout/tokens/cxLayout.ts

```diff
@@ -29,8 +31,8 @@ import { WithBordersLabels } from './cxLayoutTest';
 const Base = asLayoutToken({
   Components: {
     Helmet: cxHelmet.Default,
-    Header: cxHeader.Default,
-    Footer: cxFooter.Default,
+    Header: replaceWith(Div),
```

Comment:
I think we should change:  layoutComponents: LayoutComponents. Header & footer to be Div and then thee cxHeader and cxFooter.